include/apple2.h

```diff
@@ -83,6 +83,24 @@
 #define CH_CURS_LEFT    0x08
 #define CH_CURS_RIGHT   0x15
 
+/* These characters are not available on the ][+, but
+ * are on the //e. */
+#define CH_DEL          0x7F
+#define CH_CURS_UP      0x0B
+#define CH_CURS_DOWN    0x0A
+
+/* These are defined to be OpenApple + NumberKey */
+#define CH_F1   0xB1
+#define CH_F2   0xB2
+#define CH_F3   0xB3
+#define CH_F4   0xB4
+#define CH_F5   0xB5
+#define CH_F6   0xB6
+#define CH_F7   0xB7
+#define CH_F8   0xB8
+#define CH_F9   0xB9
+#define CH_F10  0xB0
+
```

Comment:
This isn't the no-brainer it seems to be. You're supposed to be able to write somehting like https://github.com/cc65/cc65/blob/master/libsrc/dbg/dbg.c#L106-L108 where you explicitly define replacements for missing keys. But if the defines for the keys exist then you don't know that anymore that replacements are necessary anymore.
Additionally there's code like https://github.com/FujiNetWIFI/fujinet-contiki-webbrowser/blob/main/src/apple2/contiki-conf.h#L40-L44 around that just assumes the current situation and will simply fail with duplicate defines.
Anyhow, I have right now no good idea what to do better 😒.

Reply:
Indeed... For the first example, Ctrl-H is equal to CH_DEL in fact. For the second one it would indeed require change in the target program, so I'm indeed not sure it's desirable. In my usecases I do most of my key inputs in cases, so I just do:

```
switch(c) {
    case CH_CURS_UP:
    case 'U'-'A'+1:
      ....
```

But yeah there must be a lot of programs that would break, so I suppose we can drop it and I'll just do it in my .h too :)

Reply:
Seems like a good idea!